weasyl/errorcode.py

```diff
@@ -98,6 +98,7 @@
     'noGuests': no_guest_access,
     "noCover": "No cover exists for that submission.",
     "noImageSource": "No image exists from which to create a thumbnail.",
+    "noteidInvalid": "The specified note is invalid.",
```

Comment:
I think a new error code for every invalid parameter is overkill. (It’s also not a 403.)

Reply:
Would you prefer a switch to the existing `noteRecordMissing` ("That note doesn't seem to be in our database."), and if so, would you want this to be a 404 or remain the default 422?